f5_openstack_agent/lbaasv2/drivers/bigip/listener_service.py

```diff
@@ -56,7 +56,9 @@ def create_listener(self, service, bigips):
             tls['name'] = vip['name']
             tls['partition'] = vip['partition']
 
+        network_id = service['loadbalancer']['network_id']
         for bigip in bigips:
+            self.service_adapter.get_vlan(vip, bigip, network_id)
```

Comment:
These tunnels get created as part of prep_service_networking in the call to assure_bigip_network().  The call to get_subnets_to_assure() collects a list of subnets in a service, including the subnets for members, and iterates over that list to create or validate the existence of tunnels/vlans for each.  This call is for populating the vlan list in the virtual server only so maybe this should be named differently.